### ISSUE TITLE
"Users may send as themselves: " not shown in Email Settings on high resolution screen.

### ISSUE THREAD

**Issue by dursuntosun** [salesagility/SuiteCRM#6341]
#### Issue
I could not find "Users may send as themselves: " in Email settings page when i look at on my LCD screen. When i look at mobile screen, it is exist. 
![screenshot_2](https://user-images.githubusercontent.com/22373035/45882406-c6b13580-bdb6-11e8-9a14-6d31779cfad8.png)
![screenshot_3](https://user-images.githubusercontent.com/22373035/45882482-f4967a00-bdb6-11e8-9f62-3064af2390b7.png)

#### Expected Behavior
It should be exist.

#### Steps to Reproduce

1. Open Admin page
2. Open Email Settings page
3. Look at "Users may send as themselves: " settings not there on bigger screen.
4. When you look at on mobile screen or small window, it is appear.

#### Your Environment

* SuiteCRM Version used: 10.7.9
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): Firefox 62 (64 bit)
* Environment name and version (e.g. MySQL, PHP 7): PHP 7
* Operating System and version (e.g Ubuntu 16.04): Ubuntu 16.04

**Comment by cameronblaikie** [comment]
i could not replicate this issue on Chrome or Firefox, is it possible that the "Users may send as themselves: " is on the right hand side and maybe not underneath?

**Comment by dursuntosun** [comment]
No, please look at my video;
[ScreenVideo.zip](https://github.com/salesagility/SuiteCRM/files/2414958/ScreenVideo.zip)

**Comment by cameronblaikie** [comment]
Hi @dursuntosun, you need to tick the "Use SMTP Authentication" box and then you will have those other boxes to tick.

**Comment by dursuntosun** [comment]
> 
> 
> Hi @dursuntosun, you need to tick the "Use SMTP Authentication" box and then you will have those other boxes to tick.

Yes, you are right. Bu if i don't use "Use SMTP Authentication", it should be displayed without select "Use SMTP Authentication".

**Comment by Dillon-Brown** [comment]
Hi @dursuntosun, yes you're right that you wouldn't be able to use these fields without SMTP authentication. I'm curious as to why you would want to do this though. Is there a use case where an SMTP server wouldn't require an account name and password to access? would this not make it wide open for anyone to use to send spam?
